src/libs/rover/typed_scheduler.cpp

```diff
@@ -798,11 +1034,89 @@ TypedScheduler<FloatType>::to_blueprint(Node &data)
   intensities_spatial.set(intensities);
   intensities_spatial["topology"] = "spatial_topo";
 
-  // TODO: This seems to be broken
   Node &optical_depth_spatial = fields["optical_depth_spatial"];
   optical_depth_spatial.set(optical_depth);
   optical_depth_spatial["topology"] = "spatial_topo";
 
+  // TODO: Remove this once issue #1559 is fixed
+  const bool enable_imaging_planes = rover::settings["enable_imaging_planes"].as_string() == "true";
+  if (enable_imaging_planes)
+  {
+    //
+    // Near plane mesh
+    //
+
+    write_blueprint_imaging_plane(data,
+                                  "near_plane",
+                                  near_width,
+                                  near_height,
+                                  center_near,
+                                  left,
+                                  up,
+                                  llc_near,
+                                  lrc_near,
+                                  ulc_near,
+                                  urc_near);
+
+    //
+    // View plane mesh
+    //
+
+    write_blueprint_imaging_plane(data,
+                                  "view_plane",
+                                  view_width,
+                                  view_height,
+                                  center_view,
+                                  left,
+                                  up,
+                                  llc_view,
+                                  lrc_view,
+                                  ulc_view,
+                                  urc_view);
+
```

Comment:
What is this? Does view work the same as it does in VisIt? We should make tests that render the planes to understand.